bot/resources/tags/customchecks.md

```diff
@@ -0,0 +1,21 @@
+**Custom Command Checks in discord.py**
+
+You may find yourself in need of a check decorator to do something that doesn't exist in discord.py by default, but fear not, you can make your own! Using discord.py you can use `discord.ext.commands.check` to create you own checks like this:
```

Comment:
First sentence is unnecessary. If someone wants to know how to create a check then they already have a reason for why the want to do it. In any case, the notion of having to create something because it doesn't already exist is pretty obvious and doesn't need to be explained.